newIDE/app/src/EventsSheet/index.js

```diff
@@ -544,6 +551,11 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
       label: i18n._(t`Invert Condition`),
       click: () => this._invertSelectedConditions(),
       visible: hasSelectedAtLeastOneCondition(this.state.selection),
+      accelerator: getShortcutDisplayName(
+        this.props.preferences.values.userShortcutMap[
+          'TOGGLE_CONDITION_INVERTED'
+        ] || 'KeyJ'
+      ),
```

Comment:
Was `N` as "negate" already taken?

Reply:
No, but we never use this term in the IDE so I'm not sure it's the best. N is traditionally associated to New